javalib/src/main/scala/java/lang/management/MemoryMXBean.scala

```diff
@@ -0,0 +1,42 @@
+package java.lang.management
+
+import scala.scalanative.runtime.GC
+
+trait MemoryMXBean {
```

Comment:
OpenJDK implementation - [MemoryMXBean.java](https://github.com/openjdk/jdk/blob/master/src/java.management/share/classes/java/lang/management/MemoryMXBean.java).

A few differences: 
1) The trait doesn't extend `PlatformManagedObject` 
2) The following methods aren't implemented: `getObjectPendingFinalizationCount`, `getNonHeapMemoryUsage`, `isVerbose`, `setVerbose`

Reply:
@iRevive open jdk is GPL licensed, you cannot use its sources as a reference.

Reply:
Yes, just use the Javadoc as a specification. You can also write tests that can run against the JVM and Native just to make sure the values make sense in general.